python/triton/experimental/gluon/language/amd/_layouts.py

```diff
@@ -7,9 +7,13 @@
 from triton.experimental.gluon.language._layouts import _realize_cta_layout, DistributedLayout
 from triton.experimental.gluon import language as ttgl
 
-__all__ = [
-    "AMDMFMALayout",
-]
+__all__ = ["AMDMFMALayout", "PaddedSharedLayout"]
```

Comment:
Padded shared layout is not specific to AMD so it should be placed in the common `_layouts.py`.